crates/cast/tests/data/contracts/v1/no_casm/src/lib.cairo

```diff
@@ -0,0 +1,18 @@
+#[contract]
+mod SimpleBalance {
+    struct Storage {
+        balance: felt252,
+    }
+
+    // Increases the balance by the given amount.
+    #[external]
+    fn increase_balance(amount: felt252) {
+        balance::write(balance::read() + amount);
+    }
+
+    // Returns the current balance.
+    #[view]
+    fn get_balance() -> felt252 {
+        balance::read()
+    }
+}
```

Comment:
How about an empty contract?

Reply:
What do you mean? What happens when there is an empty contract file?

Reply:
I mean there is something like an empty contract in the `cairo` repo
https://github.com/starkware-libs/cairo/blob/main/crates/cairo-lang-starknet/test_data/minimal_contract.cairo

and I wonder if we can't use it there